src/Plugin.php

```diff
@@ -116,11 +123,19 @@ protected function getEventCallback($alias, $command, $eventName)
      * @param string $alias
      * @param string $command
      * @param string $eventName
+     * @param array  $customParameters
      * @param \Phergie\Irc\Plugin\React\Command\CommandEvent $event
      * @param \Phergie\Irc\Bot\React\EventQueueInterface $queue
      */
-    public function forwardEvent($alias, $command, $eventName, Event $event, Queue $queue)
+    public function forwardEvent($alias, $command, $eventName, $customParameters, Event $event, Queue $queue)
     {
+        if (is_array($customParameters)) {
+            if (is_array($event->getCustomParams())) {
```

Comment:
Assign the return value of `$event->getCustomParams()` to a variable and use that here and below instead of calling the method twice.